app/assets/javascripts/govuk_publishing_components/accessibility-test.js

```diff
@@ -21,14 +21,27 @@
       }
 
       var violations = (typeof results === 'undefined') ? [] : results.violations
+      var incompleteWarnings = (typeof results === 'undefined') ? [] : results.incomplete
 
-      if (violations.length === 0) {
+      if (violations.length === 0 && incompleteWarnings.length === 0) {
         return callback('No accessibility issues found')
       }
 
-      var errorText = (
+      // We don't want to display incomplete warnings on preview pages
+      if (!document.querySelector(selector).parentNode.className.includes('preview')) {
```

Comment:
Can we change this so we check for a data attribute so this library doesnt know anything about our presentation classes?